src/cargo/util/toml/mod.rs

```diff
@@ -1454,7 +1454,7 @@ fn verify_lints(lints: Option<&manifest::TomlLints>) -> CargoResult<()> {
     };
 
     for (tool, lints) in lints {
-        let supported = ["rust", "clippy", "rustdoc"];
+        let supported = ["rust", "clippy", "rustdoc", "cargo"];
```

Comment:
Should we put lint system behind zflag or nightly-only in case of accidentally stabilization some lints? Though personally I want it out ASAP 😂

Alternatively we can have a field on `Lint` indicating stable/unstable or Rust version.